generator/src/main.rs

```diff
@@ -259,6 +292,12 @@ impl Parser {
                         if let Some(extends) = attr(&attributes, "extends") {
                             const EXT_BASE: i32 = 1_000_000_000;
                             const EXT_BLOCK_SIZE: i32 = 1000;
+                            let ext_number = ext_number.unwrap_or_else(|| {
+                                attr(&attributes, "extnumber")
```

Comment:
Are we relying on the caller to pass down a number for extensions, and parsing it out here instead for features? Would it be simpler/more consistent to either handle both cases (`number` vs. `extnumber`) automatically here, or alternatively to parse the number out uniformly in the caller, making the argument non-optional?

Reply:
Merging as-is for the sake of downstream.